server/server.go

```diff
@@ -559,6 +559,9 @@ func (s *Server) configureProviders() {
 	if s.globalConfiguration.DynamoDB != nil {
 		s.providers = append(s.providers, s.globalConfiguration.DynamoDB)
 	}
+	if server.globalConfiguration.ServiceFabric != nil {
```

Comment:
I think this should be 's' rather than 'server'. Happy for me to push a fix?

Reply:
oh yes, I missed that